_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,367 @@
+
+-   To reject this RequestItem you can send `{ "accept": false }` as parameters.
+
+### Combinations and usage scenarios {#readattributerequestitem-combinationsandusagescenarios}
+
+| Attribute Type | Attribute Owner | Third Party | Possible? | Automation                   | Examples/Reason                                                                                                                                                                                                                                                                                                  |
+| -------------- | --------------- | ----------- | --------- | ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          |             | N         | `N/A`                        | It makes no sense to read own IdentityAttributes.                                                                                                                                                                                                                                                                |
+| Identity       | Recipient       |             | Y         | `USER_DECISION`              | Company asks customer for its delivery address                                                                                                                                                                                                                                                                   |
+| Relationship   | Sender          |             | Y         | `USER_DECISION`              | With this combination the **sender gives the recipient the one-time permission** to write a Relationship Attribute once<br>Example: Electricity provider asks new customers for electricity meter number                                                                                                         |
+| Relationship   | Recipient       |             | Y         | `USER_DECISION`              | With this combination the **sender asks the recipient for the one-time permission** to write a Relationship Attribute<br>Example: Company asks new customer to subscribe to the newsletter and proposes the subscription as default once                                                                         |
+| Relationship   | Recipient       | Third Party | Y         | depending on confidentiality | With this combination the **sender requests a Relationship Attribute from a Relationship between the recipient and a third party. The Attribute must be owned by the recipient**<br> Example: A Social Network asks for Facebook privacy settings of a user to get senseful defaults of its own privacy settings |
```

Comment:
> depending on confidentiality

Thomas hatte ja im Confluence noch dazu geschrieben "USER DECISION /  NOT ALLOWED". Würde ich hier glaub auch machen, weil es sonst ja auch heißen könnte, dass "N/A" dastehen kann.